README.md

```diff
@@ -53,3 +53,26 @@ openshift-install destroy cluster
 Note that you almost certainly also want to clean up the installer state files too, including `auth/`, `terraform.tfstate`, etc.
 The best thing to do is always pass the `--dir` argument to `install` and `destroy`.
 And if you want to reinstall from scratch, `rm -rf` the asset directory beforehand.
+
+
+### Troubleshouting installation
+
+```Cleanup``` step should clean everything what was created during previous installation. However, sometimes installer can report error messages as ones below 
+
+``` 
+ERROR Error: Error applying plan:
+ERROR
+ERROR 3 errors occurred:
+ERROR     * module.bootstrap.aws_iam_instance_profile.bootstrap: 1 error occurred:
+ERROR     * aws_iam_instance_profile.bootstrap: Error creating IAM instance profile test-bootstrap-profile: 
+
+FATAL failed to fetch Cluster: failed to generate asset "Cluster": failed to create cluster: failed to apply using Terraform 
+``` 
+If this happen then it means that user who is trying to create new cluster has already created ```IAM``` instance profiles and creating new ones will fail.
```

Comment:
This happens when the installer tries to create an ```IAM``` instance profile with a name that matches an existing instance profile. This usually means that ```IAM``` instance profiles from a previous installation by the user didn't get deleted during an earlier cluster teardown.